docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -0,0 +1,77 @@
+# CIS Benchmarks
+
+The CIS Benchmarks represent the consensus-based effort of cybersecurity experts globally to help you protect your systems against threats more confidently.
+For more information about CIS Benchmarks check out [Center for Internet Security](https://www.cisecurity.org/cis-benchmarks)'s website.
+
+Fleet implements CIS Benchmarks using [Policies](./REST-API.md#policies). Each specific CIS benchmark check is implemented with a policy query in Fleet.
+<img src=https://user-images.githubusercontent.com/2073526/220428249-7a1b6433-24fe-4686-8dfb-b555c199f47d.png />
+
+All CIS Benchmarks implemented by Fleet are limited to a Fleet Premium or Fleet Ultimate license.
+
+The Center for Internet Security website offers documentation for all CIS Benchmarks in PDF format. Such PDFs document all the checks, their description, rationale and how to remediate them.
+
+Fleet has implemented CIS benchmarks for the following platforms:
+- CIS Apple macOS 13.0 Ventura Benchmark v1.0.0 - 11-14-2022 (82 checks) 
+- CIS Microsoft Windows 10 Enterprise Benchmark v1.12.0 - 02-15-2022 (Coming Soon!)
+
+## Manual vs Automated
+
+There are two types of CIS Benchmark checks, "Manual" and "Automated".
+- Automated: Represents recommendations for which assessment of a technical control can be fully automated and validated to a pass/fail state
+- Manual: Represents recommendations for which assessment of a technical control cannot be fully automated and requires all or some manual steps to validate that the configured state is set as expected.
+
+Fleet only implements "Automated" checks. "Manual" checks cannot be automated and/or require administrators to decide what's best for their organization's requirements.
+
+## Example CIS check
```

Comment:
Not sure if this section is necessary? Could we instead link to a policy documentation page that explains what "resolution" etc. are?

Reply:
Removed. Policies are linked in the previous section.